docs/deploymentscenarios.adoc

```diff
@@ -0,0 +1,43 @@
+[[deployment-scenarios]]
+== Deployment Scenarios 
+
+JanusGraph offers a wide choice of storage and index backends which results in great flexibility of how it can be deployed. This chapter presents a few possible deployment scenarios to help with the complexity that comes with this flexibility.
+
+Before discussing the different deployment scenarios, it is important to understand the roles of JanusGraph itself and that of the backends. First of all, applications only communicate directly with JanusGraph, mostly by sending Gremlin traversals for execution. JanusGraph then communicates with the configured backends to execute the received traversal. Since there is nothing like a _master_ JanusGraph Server, the applications can connect to any JanusGraph Server instance. They can also use a load-balancer to schedule requests to the different instances. The JanusGraph Server instances themselves don't communicate to each other directly which makes it easy to scale JanusGraph when the need arises to process more traversals.
```

Comment:
> the roles of JanusGraph itself and that of the backends. First of all, applications only communicate directly with JanusGraph, mostly by sending Gremlin traversals for execution. JanusGraph then communicates with the configured backends to execute the received traversal.

We seem to be using the term 'JanusGraph' and 'JanusGraph server' intermingly.  Can we clarify?